components/molecule/quickAction/UXDEF.md

```diff
@@ -0,0 +1,46 @@
+# SUI -Quick Action
```

Comment:
Documentation in progress (Turo)
Jira: https://jira.scmspain.com/browse/SUIC-181

Reply:
done 👍